01-messaging.md

```diff
@@ -221,8 +221,11 @@ Various fundamental types are referred to in the message specifications:
 
 * `byte`: an 8-bit byte
 * `u16`: a 2 byte unsigned integer
+* `int16`: a 2 byte signed integer
 * `u32`: a 4 byte unsigned integer
+* `int32`: a 4 byte signed integer
 * `u64`: an 8 byte unsigned integer
+* `int64`: an 8 byte signed integer
```

Comment:
Nit: `i64` matches the convention used for unsigned integers better than `int64`.